powerquery-docs/Connectors/Anaplan.md

```diff
@@ -37,9 +37,9 @@ To connect to Anaplan data:
 
 2. In the Anaplan Connector Configuration screen, enter the API and Authentication URLs:
   
-    - **Anaplan API URL**:  https://api.anaplan.com
+    - **Anaplan API URL**:  [https://api.anaplan.com](https://anaplanbulkapi20.docs.apiary.io/)
```

Comment:
Seems like this is an API link for reference, but the fix points to a help site instead.

An alternative is to do `https://api.anaplan.com` so that it's not a clickable hyperlink. 
@DougKlopfenstein what do you think?
```suggestion
    - **Anaplan API URL**:  `https://api.anaplan.com`
```